core/src/test/java/org/elasticsearch/search/sort/FieldSortBuilderTests.java

```diff
@@ -71,22 +85,26 @@ protected FieldSortBuilder mutate(FieldSortBuilder original) throws IOException
         int parameter = randomIntBetween(0, 5);
         switch (parameter) {
         case 0:
-            mutated.setNestedPath(RandomSortDataGenerator.randomAscii(mutated.getNestedPath()));
+            mutated.setNestedPath(ESTestCase.randomValueOtherThan(
+                    mutated.getNestedPath(),
+                    () -> ESTestCase.randomAsciiOfLengthBetween(1, 10)));
             break;
         case 1:
-            mutated.setNestedFilter(RandomSortDataGenerator.nestedFilter(mutated.getNestedFilter()));
+            mutated.setNestedFilter(NestedQueryBuilderGenerator.nestedFilter(mutated.getNestedFilter()));
```

Comment:
If you agree on my other comment this would then need to use ESTestCase.randomValueOtherThan.